Makefile

```diff
@@ -18,7 +18,6 @@ else
 	endif
 endif
 
-BINDATA := modules/{options,public,templates}/bindata.go
```

Comment:
This syntax does not work under `sh` which is the default shell `make` runs under so I've split it up into three individual files to stay `sh` compatible.